docs/standard/base-types/composite-formatting.md

```diff
@@ -100,7 +100,25 @@ For more information, see the <xref:System.IFormattable> and <xref:System.ICusto
 
 ### Escaping braces
 
-Opening and closing braces are interpreted as starting and ending a format item. To display a literal opening brace or closing brace, you must use an escape sequence. Specify two opening braces (`{{`) in the fixed text to display one opening brace (`{`), or two closing braces (`}}`) to display one closing brace (`}`). Braces in a format item are interpreted sequentially in the order they're encountered. Interpreting nested braces isn't supported.
+Opening and closing braces are interpreted as starting and ending a format item. To display a literal opening brace or closing brace, you must use an escape sequence. Specify two opening braces (`{{`) in the fixed text to display one opening brace (`{`), or two closing braces (`}}`) to display one closing brace (`}`).
+
+#### .NET
```

Comment:
Maybe add an intro sentence before this section saying there are some diffs between .NET/Framework.